client/app/main/skills/skillscontroller.js

```diff
@@ -10,215 +10,231 @@
+        employeeCount: 654,
+        revenue: 654
       }, {
 
-        skills: 'Gnandeep',
-        employeeCount: 'Suriseeti',
-        revenue: 'DHL'
+        skills: 'Hadoop',
+        employeeCount: 564,
+        revenue: 765
       }, {
 
-        skills: 'Nitin',
-        employeeCount: 'kondapalli',
-        revenue: 'Baylor'
-      }, ],
-      'March':[{
+        skills: 'Java',
+        employeeCount: 786,
+        revenue: 765
+      }],
+      'March': [{
 
-        skills: 'Donaldmarch',
-        employeeCount: 'Trump',
-        revenue: 'WH'
+        skills: 'Dev',
+        employeeCount: 987,
+        revenue: 765
       }, {
 
-        skills: 'Hilarymarch',
-        employeeCount: 'Clinton',
-        revenue: 'MIA'
+        skills: 'Graphic',
+        employeeCount: 986,
+        revenue: 765
 
       }, {
-        skills: 'Ahwldkhmarch',
-        employeeCount: 'kjsgckjsg',
-        revenue: 'eoihfwef'
+        skills: 'Angular',
+        employeeCount: 986,
+        revenue: 765
       }, {
 
-        skills: 'ldkhcdl',
-        employeeCount: 'podjcdp',
-        revenue: 'doihd'
+        skills: 'Jquery',
+        employeeCount: 786,
+        revenue: 764
       }, {
 
-        skills: 'Ashwini',
-        employeeCount: 'lchdohcoe',
-        revenue: 'Ash'
+        skills: 'Javascript',
+        employeeCount: 876,
+        revenue: 654
       }, {
 
-        skills: 'choecgoc',
-        employeeCount: 'Dickson',
-        revenue: 'BOFA'
+        skills: 'HTML',
+        employeeCount: 87,
+        revenue: 53
       }, {
 
-        skills: 'Vinni',
-        employeeCount: 'Jagar',
-        revenue: 'Jagaur'
+        skills: 'CSS',
+        employeeCount: 876,
+        revenue: 8
       }, {
 
-        skills: 'Anusha',
-        employeeCount: 'Manikonda',
-        revenue: 'adkchdkobc'
+        skills: 'DotNet',
+        employeeCount: 6754,
+        revenue: 64
       }, {
 
-        skills: 'Gnandeep',
-        employeeCount: 'Suriseeti',
-        revenue: 'DHL'
+        skills: 'Hadoop',
+        employeeCount: 4,
+        revenue: 64
       }, {
 
-        skills: 'Nitin',
-        employeeCount: 'kondapalli',
-        revenue: 'Baylor'
+        skills: 'Java',
+        employeeCount: 765,
+        revenue: 765
       }, ]
 
     }
+
     this.updateMonth = function(month) {
-      $scope.skillList = reports[month]
+      $scope.skillList = reports[month];
+      $scope.data = [];
+      $scope.labels = [];
+      //for loop $scope.skillList
+
+      for (var i = 0; i < $scope.skillList.length; i++) {
+        console.log("tarun")
+        // console.log($scope.data)
+      //  if ($scope.labels[i] == $scope.skillList.skills) {
+      $scope.data.push( $scope.skillList[i].revenue);
+      $scope.labels.push( $scope.skillList[i].skills);
+        console.log($scope.data)
+        }
+    //  }
     };
+
+    $scope.labels = ["Dev", "Graphic", "Angualar", "Jquery", "Javascript", "HTML", "CSS", "DotNet", "Hadoop", "Java"];
```

Comment:
Instead using these 2 lines manually, better call this.updateMonth("january")

And avoid using $scope and this, use either of those